set.mm

```diff
@@ -543393,6 +543383,26 @@ extends this idea to multiple additions (TODO).
   wl-aetr $p |- ( A. x x = y -> ( A. x x = z -> A. y y = z ) ) $=
     ( weq wal ax7 al2imi axc11 syld ) ABDZAEACDZAEBCDZAELBEJKLAABCFGLABHI $.
 
+  $( Same as ~ axc11 but with reversed antecedent.  (Contributed by NM,
+     25-Jul-2015.)  Avoid direct use of ~ ax-12 .  (Revised by Wolf Lammen,
+     30-Mar-2024.) $)
+  wl-axc11r $p |- ( A. y y = x -> ( A. x ph -> A. y ph ) ) $=
+    ( weq wal wi ax12 sps pm2.27 al2imi syld ) CBDZCEABEZLAFZCEZACELMOFCACBGHLN
+    ACLAIJK $.
+
+  ${
+    $d x y $.
+    wl-dral1v.1 $e |- ( A. x x = y -> ( ph <-> ps ) ) $.
+    $( Version of ~ dral1 with a disjoint variable condition, which does not
+       require ~ ax-13 .  Remark: the corresponding versions for ~ dral2 and
+       ~ drex2 are instances of ~ albidv and ~ exbidv respectively.
+       (Contributed by BJ, 17-Jun-2019.)  Base the proof on ~ ax12v .  (Revised
+       by Wolf Lammen, 30-Mar-2024.) $)
+    wl-dral1v $p |- ( A. x x = y -> ( A. x ph <-> A. y ps ) ) $=
+      ( weq wal nfa1 albid axc11v wi aev syl impbid bitrd ) CDFZCGZACGBCGZBDGZQ
+      ABCPCHEIQRSBCDJQDCFDGSRKCDDCDLBDCJMNO $.
```

Comment:
I see that the proof of dral1v uses axc11r instead of axc11rv https://us.metamath.org/mpeuni/dral1v.html. Shouldn't we just fix that ?

Reply:
yes, thanks